src/Pim/Component/ReferenceData/Factory/ProductValue/ReferenceDataProductValueFactory.php

```diff
@@ -107,19 +95,19 @@ protected function checkData(AttributeInterface $attribute, $data)
     /**
      * Finds a reference data by code.
      *
-     * @param AttributeInterface               $attribute
-     * @param ReferenceDataRepositoryInterface $repository
-     * @param string                           $referenceDataCode
+     * @param AttributeInterface                    $attribute
+     * @param IdentifiableObjectRepositoryInterface $repository
```

Comment:
if I understand well, that can't work
that would mean the `ReferenceDataRepositoryInterface` implements `IdentifiableObjectRepositoryInterface` 
and I'm not sure we want this constraint for the integrators